pkg/sloop/ingress/kubeclient.go

```diff
@@ -12,25 +12,27 @@ import (
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/clientcmd/api"
 )
 
 // GetKubernetesContext takes optional user preferences and returns the Kubernetes context in use
-func GetKubernetesContext(masterURL string, kubeContextPreference string) (string, error) {
+func GetKubernetesContext(masterURL string, kubeContextPreference string, privilegedAccess bool) (string, error) {
 	glog.Infof("Getting k8s context with user-defined config masterURL=%v, kubeContextPreference=%v.", masterURL, kubeContextPreference)
+	contextInUse := kubeContextPreference
```

Comment:
It seems unnecessary as the value is assigned anyways on line 32

Reply:
Reason I have initialized that way is that : If privileged access not given then it directly assing kubecontextpreference

Reply:
@sana-jawad  This is the latest file